vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ProjectDetailsComponent.java

```diff
@@ -114,14 +117,21 @@ private void initFormFields() {
     experimentalDesignToggleComponent = new ToggleDisplayEditComponent<>(Span::new, new TextArea(),
         createPlaceHolderSpan());
     speciesMultiSelectComboBox = new MultiSelectComboBox<>();
+    speciesMultiSelectComboBox.setClearButtonVisible(false);
     specimenMultiSelectComboBox = new MultiSelectComboBox<>();
+    specimenMultiSelectComboBox.setClearButtonVisible(false);
```

Comment:
Why was the clear button removed? 
I'd argue it's fine for the user to clear all the values specified as long as a new value is provided

Reply:
we cannot remove species, specimens and analytes from experiments. Thus this should be discouraged. Removals are ignored in the application.